api/http/handler/webhooks/webhook_create.go

```diff
@@ -0,0 +1,64 @@
+package webhooks
+
+import (
+	"net/http"
+
+	"github.com/asaskevich/govalidator"
+	"github.com/portainer/portainer"
+	httperror "github.com/portainer/portainer/http/error"
+	"github.com/portainer/portainer/http/request"
+	"github.com/portainer/portainer/http/response"
+	"github.com/satori/go.uuid"
+)
+
+type webhookCreatePayload struct {
+	ServiceID  string
+	EndpointID int
+}
+
+func (payload *webhookCreatePayload) Validate(r *http.Request) error {
+	if govalidator.IsNull(payload.ServiceID) {
+		return portainer.Error("Invalid ServiceID")
+	}
+	if govalidator.IsNull(string(payload.EndpointID)) {
+		return portainer.Error("Invalid EndpointID")
+	}
+	if payload.EndpointID < 1 {
```

Comment:
I don't think that we need this validation. The UI should never pass an identifier with a value that is less that 1. Plus, it would trigger an endpoint not found error, which is fine.